mrmustard/physics/gaussian.py

```diff
@@ -16,6 +16,8 @@
 This module contains functions for performing calculations on Gaussian states.
 """
 
+import tensorflow as tf
```

Comment:
This should not be imported here. Any new math functionality that you need should go through the interface:
1. add the required new methods to the [math interface](https://github.com/XanaduAI/MrMustard/blob/main/mrmustard/math/math_interface.py)
2. implement them in the [tf backend](https://github.com/XanaduAI/MrMustard/blob/main/mrmustard/math/tensorflow.py)
3. use them here via `math.your_method`

Reply:
Thanks @ziofil! This will be implemented as you describe. I've done it this way to test that it works as expected... polishing details to come 🧹